README.md

```diff
@@ -18,11 +47,21 @@ Nx is designed to help you create and build enterprise grade Angular application
 
 ## Generate your first application
 
-Run `ng generate app myapp` to generate an application. When using Nx, you can create multiple applications and libraries in the same CLI workspace. Read more [here](http://nrwl.io/nx).
-
+Run `ng generate app myapp` to generate an application. When using Nx, you can create multiple applications and libraries in the same CLI workspace. Read more [here](http://nrwl.io/nx). -->
+<!-- 
 ## Development server
 
-Run `ng serve --project=myapp` for a dev server. Navigate to `http://localhost:4200/`. The app will automatically reload if you change any of the source files.
+Run `ng serve --project=myapp` for a dev server. Navigate to `http://localhost:4200/`. The app will automatically reload if you change any of the source files. -->
+
+## The Goal of Angular CLI
```

Comment:
I do not think we need to tell what is the goal of Angular CLI because it is not related to our project.